blueprint/api.apib

```diff
@@ -726,114 +854,207 @@ A user is able to have multiple roles through their user roles.
 
     + Headers
 
-            Accept: application/json
+            Accept: application/vnd.api+json
 
-+ Response 200 (application/json)
++ Response 200 (application/vnd.api+json; charset=utf-8)
 
   + Attributes (Roles Response)
 
-# Group Role Skills
-
-This resource identifies a relationship between a Role and a Skill. For example, the skill of "JavaScript" may relate to the role of "Backend Developer," and also "Front End Developer" and even "Mobile Developer".
+## Role [/roles/{id}]
 
-We can use the many skills grouped under a role to suggest popular skills to users who may have self-identifed as a "Backend Developer".
+### Retrieve a role [GET]
 
-## Role Skill [/role-skills]
++ Parameters
 
-### Create a role skill [POST]
+    + id (number) - Id of a role.
 
 + Request
 
     + Headers
 
-            Accept: application/json
+            Accept: application/vnd.api+json
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Role Response)
+
++ Response 404 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Record Not Found Response)
+
+# Group Role Skills
+
+This resource identifies a relationship between a Role and a Skill. For example, the skill of "JavaScript" may relate to the role of "Backend Developer," and also "Front End Developer" and even "Mobile Developer".
```

Comment:
This is a little awkward. It almost sounds like the relationships are variable in suggestion. I get the reason, but maybe just list them out? 

> ... role of "Backend", "Frontend" and "Mobile"

This is more of an opinion, so take it as you will.